bundle/edu.gemini.pit/src/main/scala/edu/gemini/pit/ui/robot/ProblemRobot.scala

```diff
@@ -768,24 +768,6 @@ case class TimeProblems(p: Proposal, s: ShellAdvisor) {
     case _ => None
   }
   lazy val b3ReqOrZero: TimeAmount = b3Req.map(_.time).getOrElse(TimeAmount.empty)
-  lazy val jointNotAllowed: List[Problem] = {
```

Comment:
We have often kept these because they tend to be reused in future semesters. We could instead make `jointProposalNotAllowed` empty

Reply:
Ok I'll add it back and comment it out.